ferrocene/doc/user-manual/src/rustfmt/cli.rst

```diff
@@ -8,23 +8,37 @@
 
    .. cli:option:: --check
 
-      Run in 'check' mode. Exits with 0 if input is formatted correctly. Exits with 1 and prints a diff if formatting is required.
+      Run in 'check' mode.
 
-   .. cli:option:: --emit [files|stdout|coverage|checkstyle|json]
+      Exits with status code 0 if input is formatted correctly.
+      Exits with status code 1 and prints a diff if formatting is not correct.
 
-      What data to emit and how
+   .. cli:option:: --emit [files|stdout]
+
+      What kind of output to emit.
+
+      - ``files``, which emits formatted output to files, is the default
+      - ``stdout`` emits the formatted output to stdout
 
    .. cli:option:: --backup
 
       Backup any modified files.
 
+      This copies the files before they are unmodified in same directory as the original files,
+      and changes the file extension to ``.bk``, replacing the ``.rs`` extension.
+
    .. cli:option:: --config-path
 
-      Recursively searches the given path for the rustfmt.toml config file. If not found reverts to the input file path
+      Recursively searches the given path for the ``rustfmt.toml`` or ``.rustfmt.toml`` config file.
+      If not found, it reverts to the default behavior of searching the parent directories of the
+      file to be formatted.
 
    .. cli:option:: --edition [2015|2018|2021]
 
-      Rust edition to use
+      Rust edition to use, which defaults to 2015 edition.
```

Comment:
Does it? I assumed 2021 aka. the most recent one

Reply:
No, all rustc-related tools assume the 2015 edition if no edition flag is specified. Otherwise every published edition would result in a breaking change of the `rustc foo.rs` behavior.